pkg/ui/styl/layout/navigation-bar.styl

```diff
@@ -97,6 +97,7 @@ $subnav-underline-height = 3px
         display inline
         padding 0
         text-decoration none
+        color inherit
```

Comment:
Hm, I thought it would inherit or "cascade" by default…

Reply:
css 🤦‍♂️